features/graph/dao/api/src/main/java/org/opennms/netmgt/topology/VertexEntity.java

```diff
@@ -31,8 +31,16 @@
 import javax.persistence.DiscriminatorValue;
 import javax.persistence.Entity;
 
+import org.opennms.netmgt.graph.dao.api.EntityProperties;
+
 @Entity
 @DiscriminatorValue("vertex")
 public class VertexEntity extends AbstractGraphEntity {
 
+    // TODO: Patrick: discuss with mvr: we reuse "id" for 2 different things: databaseid and an identifier (part of VertexRef)
+    // we should come up with a better naming to reduce confusion?
```

Comment:
@mvrueden , lets discuss

Reply:
I agree. However, this is only used in tests and should not be used by the persistence in any way.
It could be rewritten to something like `vertex.getProperty(EntityProperties.ID)` in general.

The naming is a bit confusing, but so far I don't have a better option. I am all for alternatives :)

Reply:
we could add a comment to make this clear.
Just to make it clear for me: 
you say the id in the properties is a domain specific identifier and not a database id, correct?
What about the id in a graph? Isn't the namespace already domain identifier and thus a graph shouldn't have an id?

Reply:
ad discussed: we leave namespace as an attribute, the id as a property. Database ids are renamed to dbId.